docs/api-testing/composer.md

```diff
@@ -1,77 +1,91 @@
 :::
 
-## Composer UI
-
-Use the following image as a reference for the numbered items listed below:
-
-<img src={useBaseUrl('img/api-fortress/2021/01/testComposerPage-annotated.png')} alt="Test Composer Page Annotated"/>
-
-1. __Add Request__ / __Assertions__: This button displays all available components.
-   If a component is not valid for the operation you are conducting, it will not be made available to help avoid mistakes.
-   For instance, if you don’t add a `POST` first, you cannot add a `POST` Body or `POST` Param.
+## Transform Component
 
-   <img src={useBaseUrl('img/api-fortress/2019/06/components.png')} alt="Test Components"/>
+Transforms an existing component into another component of the same type.<br/>
+<img src={useBaseUrl('img/api-fortress/2021/01/transformComponent.png')} alt="Add Component"/>
 
-  :::note
-  Free trial accounts do not give you access to all available components.
-  :::
+## Delete Component
+Deletes a selected component from the test while using Visual view.<br/>
+<img src={useBaseUrl('img/api-fortress/2021/01/deleteComponent.png')} alt="Delete Component"/>
 
-2. __Transform Component__: This button allows you to easily transform an existing component into another component of the same type.
+## Invoke Snippet
+Allows you to use a previously created code snippet stored in [The Vault](/api-testing/on-prem/quick-start/the-vault).<br/>
+<img src={useBaseUrl('img/api-fortress/2021/01/invokeSnippet.png')} alt="Invoke Snippet"/>
 
-3. __Delete Component__: This button allows you to delete a selected component from the test while using visual mode.
+## Export Snippet
+Allows you to export a selected code snippet to the vault in order to be re-used later, or in another test.<br/>
+<img src={useBaseUrl('img/api-fortress/2021/01/exportSnippet.png')} alt="Export Snippet"/>
 
-4. __Invoke Snippet__: This button allows you to use a previously created code snippet stored in [The Vault](/api-testing/on-prem/quick-start/the-vault).
+## Save Test
+Saves your progress.<br/>
+<img src={useBaseUrl('img/api-fortress/2021/01/saveTest.png')} alt="Save Test"/>
 
-5. __Export Snippet__: This button allows you to export a selected code snippet to the vault in order to be re-used later, or in another test.
+## Run Test
+Executes a test.<br/>
+<img src={useBaseUrl('img/api-fortress/2021/01/runTest.png')} alt="Run Test"/>
 
-6. __Save Test__: This button saves your progress.
+## Input Sets
+Displays the Input Set view where you can store input data sets to reuse across all your tests.<br/>
+<img src={useBaseUrl('img/api-fortress/2021/01/inputSets.png')} alt="Input Sets" width="500"/>
 
-7. __Run Test__: This button executes a test.
+There are two types of input data sets you can use:
+* __Global Parameters__: variables that are available across all tests in the project. Reference these variables simply by calling it within the test using the convention `“${VARIABLE}”`.
```

Comment:
Global Parameters are variables that are available to be used throughout a test.